EXPO/screens/UserInfoDBManager.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import  SQLite   from 'react-native-sqlite-storage';
+import { localDB } from '../screens/Constants';
+
+
+export default  class UserInfoDBManager extends Component {
+
+  constructor() {
+       this.sqlite = SQLite;
```

Comment:
fix indentation

Reply:
UserInfoDBManager.js file has been deleted.No used in App